daemon/core/api/tlv/corehandlers.py

```diff
@@ -2042,6 +2042,7 @@ def __init__(self, request, client_address, server):
         }
```

Comment:
this CoreUdpHandler class inherits from CoreHandler but does not invoke its __init__() method, so the `self.coreemu` member never gets set.